### PR TITLE
Add docs and scripts for configuring edge to work on Linux devices with IPv6 networks

### PR DESCRIPTION
**Pull request by shantanu1singh**
Add docs and scripts for configuring edge to work on Linux devices with IPv6 networks

### CODE REVIEW

File: scripts/linux/configureDockerIPv6.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+[ -z "$3" ] && echo "Please specify the network interface" && exit 1
+
+DOCKER0_BRIDGE_SUBNET=$1
+IOT_EDGE_SUBNET=$2
+NETWORK_INTERFACE=$3
+
+echo "Installing jq and grep..."
+apt-get update -y
+apt-get install -y jq grep
+echo "Jq and grep installed."
```
Comment (arsing) [inline scripts/linux/configureDockerIPv6.sh]: `Jq` -> `jq`

File: scripts/linux/configureDockerIPv6.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+echo "Restarting docker..."
+systemctl restart docker
+echo "docker restarted."
+
+echo "Running ndppd configuration script."
+chmod +x ./installNdppd.sh
+sh ./installNdppd.sh $DOCKER0_BRIDGE_SUBNET $IOT_EDGE_SUBNET $NETWORK_INTERFACE
+echo "ndppd configuration script executed."
+
```
Comment (arsing) [inline scripts/linux/configureDockerIPv6.sh]: Remove excess newline at end of file.

File: scripts/linux/installNdppd.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  rule $IOT_EDGE_SUBNET {
+    auto
+  }
+}
+EOF
+echo "Configured ndppd."
+
+echo "Restarting ndppd..."
+systemctl restart ndppd
+echo "ndppd restarted."
```
Comment (arsing) [inline scripts/linux/installNdppd.sh]: Missing newline at end of file.

File: scripts/linux/configureDockerIPv6.sh [hunk 4]
```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```
Comment (arsing) [inline scripts/linux/configureDockerIPv6.sh]: This is a script, so make it `+x` (0755)

File: scripts/linux/installNdppd.sh [hunk 5]
```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```
Comment (arsing) [inline scripts/linux/installNdppd.sh]: This is a script, so make it `+x` (0755)

File: scripts/linux/configureDockerIPv6.sh [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
```
Comment (arsing) [inline scripts/linux/configureDockerIPv6.sh]: Add `set -euo pipefail`

File: scripts/linux/installNdppd.sh [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
```
Comment (arsing) [inline scripts/linux/installNdppd.sh]: Add `set -euo pipefail`

**Comment by arsing** [comment]
@veyalla Do you want to review the docs?

**Comment by ebertrams** [comment]
Looks good to me. Thanks @shantanu1singh !

**Pull request merged**